amundsen_application/static/js/components/Footer/index.tsx

```diff
@@ -29,8 +30,7 @@ export class Footer extends React.Component<FooterProps> {
   }
 
   generateDateTimeString = () => {
-    // 'moment.local' will utilize the client's local timezone.
-    return moment.unix(this.props.lastIndexed).local().format('MMMM Do YYYY [at] h:mm:ss a');
+    return formatEpochTime(this.props.lastIndexed, DATE_FORMAT_LONG);
```

Comment:
@ttannis is there something I should be doing here with local timezones?

Reply:
Depends on the logic of `formatEpochTime`. The idea here is that users must see this time according to their local timezone to avoid confusion. For example if the actual time was 6AM pacific time, users in New York should ideally see this as 9AM eastern time.

Reply:
I'm just using the `moment` constructor. I tested it while spoofing my location and it appears to update the timezone. Maybe the `moment.unix` function we were using before was trying to reset to GMT or something.

Reply:
Ah, so I think this was to account for daylight savings time as well.  I think the original logic could be summarized as "convert to GMT then transform to local timezone".  Whatever methods we use here just has to account for that so people don't get confused.

Reply:
@danwom so does this new method account for daylight saving time and local timezones?

Reply:
Timezones should be accounted for in this version.